contrib/time_series_expectations/time_series_expectations/expectations/prophet_model_deserializer.py

```diff
@@ -0,0 +1,29 @@
+# Suppress an annoying warning from Prophet.
+import logging
+
+logger = logging.getLogger("prophet.plot")
+logger.setLevel(logging.CRITICAL)
+try:
+    from prophet import Prophet
+    from prophet.serialize import model_from_json
+except ImportError:
+    logger.warning(
+        "Unable to import Prophet. Please install prophet to use this Expectation."
+    )
+    model_from_json = None
+
+
+class ProphetModelDeserializer:
+    """A class for deserializing a Prophet model from JSON"""
+
+    def get_model(
```

Comment:
Yes, in the future, I think we end up with several of these classes for different types of time series models. I thought about fleshing out that class structure in this PR, but it would have resulted in a lot of code smells.